crates/core/src/models/transaction.rs

```diff
@@ -94,7 +94,8 @@ impl ConvertibleStarknetTransaction for StarknetTransaction {
         let to = tx.to();
 
         let v = if signature.odd_y_parity { 1 } else { 0 } + 35 + 2 * CHAIN_ID;
-        let signature = Some(Signature { r: signature.r, s: signature.s, v: U256::from_limbs_slice(&[v]) });
+        let signature =
+            Some(Signature { r: signature.r, s: signature.s, v: U256::from_limbs_slice(&[v]), y_parity: None });
```

Comment:
what is this y_parity new item?

Reply:
They've added this [here](https://github.com/paradigmxyz/reth/blob/07a1af85ea797df30ef3a82ab51453ddf6411563/crates/rpc/rpc-types/src/eth/transaction/signature.rs#L21-L23), I'm not sure exactly what we should put here and as it's an option I put it `None` for now. Should I make a TODO just in case?

Reply:
can you create an issue to investigate this value? it can be labeled a `timebox-3hr`
the point would be to check whether we need to set it or smth,
letting it as none for now is ok

Reply:
y parity is linked to v, basically (since [eip-155](https://eips.ethereum.org/EIPS/eip-155)), v = 2 * CHAIN_ID + 35 or 2 * CHAIN_ID + 36 depending on the parity of the point on the elliptic curve.

Reply:
does that imply something? here we set parity to None, always
maybe it should be set somehow?

can be delayed in another issue (the one we mentioned together in daily)